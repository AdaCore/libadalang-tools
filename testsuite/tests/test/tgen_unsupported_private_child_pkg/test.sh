#! /usr/bin/env bash

gnattest -P test.gpr --gen-test-vectors
