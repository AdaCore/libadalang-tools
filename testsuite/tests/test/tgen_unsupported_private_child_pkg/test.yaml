description:
  Test that gnattest detects private child packages when using tgen and skips
  them correctly.

driver: shell_script
