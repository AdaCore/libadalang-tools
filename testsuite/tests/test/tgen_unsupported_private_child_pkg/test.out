gnattest-tgen: Pkg.Child: Test input generation not supported for private packages; skipping
Units remaining: 3     gnattest: Pkg.Child is private or child of private; skipping
Units remaining: 2     Units remaining: 1     
